SU-HSDO/suhumsci
# fix(STN-443): Sprint 20 QA

# READY FOR REVIEW

## Summary
Sprint 20 QA Feedback fixes.

## Steps to Test
1. `npm run test` passes
2. run backstop js to make sure not too much has changed, and nothing has broken

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
